ci-operator/step-registry/openshift/e2e/openstack/proxy/openshift-e2e-openstack-proxy-workflow.yaml

```diff
@@ -0,0 +1,31 @@
+workflow:
+  as: openshift-e2e-openstack-proxy
+  steps:
+    pre:
+      - chain: ipi-openstack-pre
+    test:
+      - ref: openshift-e2e-test
+    post:
+      - chain: ipi-openstack-post
+    env:
+      CONFIG_TYPE: "proxy"
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      TEST_SUITE: experimental/reliability/minimal
+      # These tests fail when we use HTTP_PROXY / HTTPS_PROXY to reach the OCP cluster and run tests against it.
+      # This will be investigated later, but in the meantime we skip them to be able to test the rest and
+      # provide good coverage.
+      TEST_SKIPS: >-
```

Comment:
Note: @stbenjam has an approach in PR [26389](https://github.com/openshift/origin/pull/26398) to directly exclude the proxy related tests from within the e2e test suites